pros/cli/upload.py

```diff
@@ -75,7 +75,7 @@ def upload(path: str, port: str, **kwargs):
         logger(__name__).debug(f"Invalid target provided: {kwargs['target']}")
         logger(__name__).debug('Target should be one of ("v5" or "cortex").')
     if not port:
-        return -1
+        raise click.UsageError('No port provided or located. Make sure to specify --target if needed.')
```

Comment:
Can you wrap this with `dont_send(click.UsageError(....))` so that I don't see reports of this in Sentry.

Reply:
Good shout